web/themes/new_weather_theme/assets/sass/components/radar.scss

```diff
@@ -58,15 +56,29 @@
     }
   }
 }
+  
+.wx-radar-container.wx-radar-container__expanded {
+  aspect-ratio: 3/5 !important;
+  height: auto !important;
+  width: auto !important;
+}
 
 @include at-media("tablet") {
   .wx-radar-container {
-    &.wx-radar-container__expanded {
-      aspect-ratio: 1/1;
-    }
+      aspect-ratio: 3/4 !important;
   }
```

Comment:
Mistake on my part, this should have been 4/3 so it's wider than it is tall in the collapsed state. Then 3/4 should be the ratio for the expanded, so then it becomes taller.